tasks/uk/census/scotland.py

```diff
@@ -0,0 +1,56 @@
+# http://www.scotlandscensus.gov.uk/ods-web/data-warehouse.html#bulkdatatab
+
+import csv
+import os
+import re
+
+from luigi import Parameter
+
+from tasks.util import shell, DownloadUnzipTask, TempTableTask
+from tasks.meta import current_session
+
+
+class DownloadScotlandLocal(DownloadUnzipTask):
+
+    URL = 'http://www.scotlandscensus.gov.uk/ods-web/download/getDownloadFile.html?downloadFileIds=Output%20Area%20blk'
+
+    def download(self):
+        shell('wget -O {output}.zip {url}'.format(output=self.output().path, url=self.URL))
+
+
+class ImportScotland(TempTableTask):
+    table = Parameter()
+
+    def requires(self):
+        return DownloadScotlandLocal()
+
+    @staticmethod
```

Comment:
Why using a static method instead of a top level function?

Reply:
It's different per-task. The idea is to have a method to normalize column names, but the normalization is different for Scotland and UK. I will probably create an abstract class to make it more obvious.